app/init.php

```diff
@@ -111,7 +111,7 @@
 const APP_KEY_ACCCESS = 24 * 60 * 60; // 24 hours
 const APP_USER_ACCCESS = 24 * 60 * 60; // 24 hours
 const APP_CACHE_UPDATE = 24 * 60 * 60; // 24 hours
-const APP_CACHE_BUSTER = 405;
+const APP_CACHE_BUSTER = 4051;
```

Comment:
on purpose? 👀

Reply:
I always use the console version for the cache buster, but it was already at 405, so I added a 1 like 4.0.5.1 🤣